### PR TITLE
add custom passlog directory when passlogfile is setted

### PR DESCRIPTION
**Pull request by nsilberman**
sometimes you can't write within the current working dir

### CODE REVIEW

File: src/main/java/net/bramp/ffmpeg/builder/FFmpegBuilder.java [hunk 1]
```diff
@@ -56,6 +56,7 @@ public String toString() {
   // Global Settings
   boolean override = true;
   int pass = 0;
+  String pass_directory = "./";
```
Comment (bramp) [inline src/main/java/net/bramp/ffmpeg/builder/FFmpegBuilder.java]: Could we leave the default the empty string to preserve identical behavior.

File: src/main/java/net/bramp/ffmpeg/builder/FFmpegBuilder.java [hunk 2]
```diff
@@ -240,7 +246,7 @@ public FFmpegOutputBuilder addStdoutOutput() {
       args.add("-pass", Integer.toString(pass));
 
       if (pass_prefix != null) {
-        args.add("-passlogfile", pass_prefix);
+        args.add("-passlogfile").add(pass_directory + pass_prefix);
```
Comment (bramp) [inline src/main/java/net/bramp/ffmpeg/builder/FFmpegBuilder.java]: I don't think you need to chain the adds,  as add takes multiple arguments.

**Comment by nsilberman** [comment]
updated and squashed

**Comment by nsilberman** [comment]
:)

**Pull request merged**